drools-ansible-rulebook-integration-api/src/main/java/org/drools/ansible/rulebook/integration/api/domain/Rule.java

```diff
@@ -1,54 +1,70 @@
 package org.drools.ansible.rulebook.integration.api.domain;
 
-import java.util.List;
-import java.util.concurrent.atomic.AtomicInteger;
-
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import org.drools.ansible.rulebook.integration.api.RuleConfigurationOption;
+
 import org.drools.ansible.rulebook.integration.api.RuleConfigurationOptions;
 import org.drools.ansible.rulebook.integration.api.domain.actions.Action;
 import org.drools.ansible.rulebook.integration.api.domain.actions.MapAction;
 import org.drools.ansible.rulebook.integration.api.domain.conditions.AstCondition;
 import org.drools.ansible.rulebook.integration.api.domain.conditions.Condition;
 import org.drools.ansible.rulebook.integration.api.domain.temporal.Throttle;
 import org.drools.ansible.rulebook.integration.api.domain.temporal.TimeWindowDefinition;
-import org.drools.ansible.rulebook.integration.api.rulesengine.RulesExecutionController;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Rule {
     private boolean enabled;
 
-    private final RuleGenerationContext ruleGenerationContext = new RuleGenerationContext();
+    final RuleGenerationContext ruleGenerationContext = new RuleGenerationContext();
```

Comment:
Nitpick: let's keep this `private`. I know I'm an old style Java developer but I don't like accessing fields directly from outside the owner class :)